include/bluetooth/mesh/vnd/dm_cli.h

```diff
@@ -0,0 +1,225 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/**
+ * @file
+ * @defgroup bt_mesh_dm_cli Distance Measurement Client model
+ * @{
+ * @brief API for the Distance Measurement Client model.
+ */
+
+#ifndef BT_MESH_DM_CLI_H__
+#define BT_MESH_DM_CLI_H__
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include <bluetooth/mesh/models.h>
+#include <zephyr/bluetooth/addr.h>
+#include <zephyr/sys/slist.h>
+#include <zephyr/bluetooth/mesh/access.h>
+#include "dm_common.h"
+#include <dm.h>
+
+struct bt_mesh_dm_cli;
+
+/** Parameters for the Distance Measurement Result Status message. */
+struct bt_mesh_dm_cli_result {
```

Comment:
I still want result<ins>s</ins>, really. There are N measurement results here, not one.